docker/nonvoting_mixnet/conf/mix2/mix_data/identity.public.pem

```diff
@@ -1,3 +1,4 @@
------BEGIN ED25519 PUBLIC KEY-----
-1x8tq04hY+i83o9Yn2nrfTkFj4jXIMWCWilR7fgrWyM=
------END ED25519 PUBLIC KEY-----
+-----BEGIN ED25519 SPHINCS+ PUBLIC KEY-----
```

Comment:
hmmmmmm. I wonder if it would make sense to keep the test data for `eddsa` so it's still technically possible to run the docker tests without the pq stuff?

Reply:
i don't think we want to accumulate more pem files

Reply:
also it's not possible to run docker with the old keying unless you change cert.go to use the old scheme

Reply:
OK, fair enough.